pkg/cmd/cli/cmd/deploy.go

```diff
@@ -369,13 +391,26 @@ func (o DeployOptions) reenableTriggers(config *deployapi.DeploymentConfig, out
 		}
 	}
 	if len(enabled) == 0 {
-		fmt.Fprintln(out, "No image triggers found to enable")
+		fmt.Fprintln(o.out, "No image triggers found to enable")
 		return nil
 	}
 	_, err := o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err != nil {
 		return err
 	}
-	fmt.Fprintf(out, "Enabled image triggers: %s\n", strings.Join(enabled, ","))
+	fmt.Fprintf(o.out, "Enabled image triggers: %s\n", strings.Join(enabled, ","))
 	return nil
 }
+
+func (o DeployOptions) getLogs(config *deployapi.DeploymentConfig) error {
+	opts := deployapi.DeploymentLogOptions{
+		Follow: true,
+	}
+	readCloser, err := o.osClient.DeploymentLogs(config.Namespace).Get(config.Name, opts).Stream()
```

Comment:
Currently it just wait until the replication controller is created. I will open a separate issue to make it wait for the deployer pod to come up.